internal/pkg/usage/usage.go

```diff
@@ -23,17 +23,20 @@ Usage: %s [options]
 Server Options:
     -r, --registry                  Indicates service should use Registry
     -p, --profile <name>            Indicate configuration profile other than default
-    -confdir                        Specify local configuration directory
+    --confdir                       Specify local configuration directory
 Common Options:
     -h, --help                      Show this message
 `
 
 var configSeedUsageStr = `
 Usage: %s [options]
 Server Options:
+    -c, --cmd <dir>                 Provide absolute path to "cmd" directory containing EdgeX services
```

Comment:
I think this reads a bit cleaner as 
```
Provide absolute path to "cmd" directory containing EdgeX service configuration
```

Reply:
@anonymouse64 Change applied. Need re-approval due to rebase.